convert2rhel/utils.py

```diff
@@ -501,21 +501,35 @@ def download_pkg(
         loggerinst.warning("Output from the yumdownloader call:\n%s" % (output))
 
-        if not "CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK" in os.environ:
-            loggerinst.critical(
-                "Couldn't download the %s package. This means we will not be able to do a"
-                " complete rollback and may put the system in a broken state.\n"
-                "Check to make sure that the %s repositories are enabled"
-                " and the package is updated to its latest version.\n"
-                "If you would rather ignore this check set the environment variable"
-                " 'CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK'." % (pkg, system_info.name)
-            )
+        # add in the new envar that should be changed to along with the old one
+        allow_old_envar_names = ("CONVERT2RHEL_INCOMPLETE_ROLLBACK", "CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK")
+
+        # search for either envar in the system to see if it has been set
+        if any(envvar in os.environ for envvar in allow_old_envar_names):
+
+            # check to see which envar was used
+            if "CONVERT2RHEL_INCOMPLETE_ROLLBACK" not in os.environ:
+                # give the warning about the old envar not being used anymore if it is uesed
+                loggerinst.warning(
+                    "You are using the deprecated 'CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK'"
+                    " environment variable.  Please switch to 'CONVERT2RHEL_INCOMPLETE_ROLLBACK' instead."
+                )
+
+                # give regular warning if the new envar is used
+                loggerinst.warning(
+                    "Couldn't download the %s package. This means we will not be able to do a"
+                    " complete rollback and may put the system in a broken state.\n"
+                    "'CONVERT2RHEL_INCOMPLETE_ROLLBACK' environment variable detected, continuing conversion." % (pkg)
+                )
         else:
-            loggerinst.warning(
-                "Couldn't download the %s package. This means we will not be able to do a"
-                " complete rollback and may put the system in a broken state.\n"
-                "'CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK' environment variable detected, continuing conversion."
-                % (pkg)
+            loggerinst.critical(
+                "Couldn't back up the packages: %s. This means that if a rollback is needed,"
+                "there is no guarantee that the packages will be restored on rollback, which"
+                "could put the system in a broken state.\n Check to ensure that the %s "
+                "repositories are enabled, and the packages are updated to their latest "
+                "versions.\n If this error still occurs after re-running the conversion, "
+                "you can set the environment variable CONVERT2RHEL_INCOMPLETE_ROLLBACK = 1"
+                "to ignore this check." % (pkg, system_info.name)
```

Comment:
```suggestion
                "Couldn't back up the packages: %s. This means that if a rollback is needed,"
                "there is no guarantee that the packages will be restored on rollback, which"
                "could put the system in a broken state.\nCheck to ensure that the %s "
                "repositories are enabled, and the packages are updated to their latest "
                "versions.\nIf this error still occurs after re-running the conversion, "
                "you can set the environment variable 'CONVERT2RHEL_INCOMPLETE_ROLLBACK=1'"
                "to ignore this check." % (pkg, system_info.name)
```

Reply:
I will go ahead and get these changes added in! Thank you for all of the comments with this!

Reply:
This wasn't added

Reply:
ahh I know what happened, I had stashed some changes and lost some of them. After trying to retrieve the changes, I lost progress and had to restart. So I must have missed this. I will check to be sure I got everything else back as well.